packages/wrangler/src/index.tsx

```diff
@@ -527,6 +527,12 @@ function createCLIParser(argv: string[]) {
 						describe: "Don't actually publish",
 						type: "boolean",
 					})
+					.option("keep-vars", {
+						describe:
+							"Stop wrangler from deleting vars that are not present in the wrangler.toml",
```

Comment:
I think this needs to be expanded a bit more.

Reply:
It seems rather clear to me, what could be expanded upon? I could just have context and am making assumption of knowledge here.

Reply:
Something like:

> Stop Wrangler from deleting vars that are not present in the wrangler.toml.
> By default Wrangler will remove all vars and replace them with those found in the `wrangler.toml` configuration.
> If your development approach is to modify vars after deployment via the dashboard you may wish to set this flag.